install/install.sh

```diff
@@ -395,7 +407,7 @@ else
   VERBOSE=
 fi
 
-echo
+printf '\n'
```

Comment:
This is nonsense, a simple `echo` will always work here, and spurious `printf` is penalising the shells that don’t have it as built-in utility.

`echo` is portable iff…
 - its first argument does not begin with a hyphen-minus `-`
 - there is no backslash in any of its arguments (not even escaped)

Reply:
This is an install script; not something that is executed constantly; optimising for performance is not the goal here. The goal is to be portable and posix compliant. If we mix echos and printf based on the exact conditions under which we believe are portable, then the next person that comes along and once to make a change will need to be armed with the same knowledge.

I'd vote to keep things consistent and simple; especially for an installation script that is run once per version (at most). I don't think the milliseconds saved per release with a builtin is worth the complexity.